.travis.yml

```diff
@@ -9,14 +9,16 @@ env:
   global:
     - RUN_PHPSTAN="FALSE"
   matrix:
-    - PREFER_LOWEST="" WITH_COVERAGE="--coverage-clover=coverage.xml"
-    - PREFER_LOWEST="--prefer-lowest" $WITH_COVERAGE=""
```

Comment:
Also gets rid of this "random" `$`

Reply:
This was a ENV var before I guess

Reply:
I guess it did not matter, and still does not matter - either way `WITH_COVERAGE` ended up being the empty string.